client/jetpack-cloud/sections/partner-portal/controller.tsx

```diff
@@ -83,9 +83,10 @@ export function licensesContext( context: PageJS.Context, next: () => void ): vo
 }
 
 export function issueLicenseContext( context: PageJS.Context, next: () => void ): void {
+	const { site_id } = context.query;
```

Comment:
Can we rename the `site_id` variable to `siteId`? In practice, this would look like this:
```
const { site_id: siteId } = context.query; // we rename site_id to siteId
```

Reply:
I forgot to mention that on Calypso, the convention is to use camel case as much as possible.

Reply:
I've updated this and other instances from site_id to siteId